### PR TITLE
RA-1831: "@Override" should be used on overriding and implementing methods

### PR DESCRIPTION
**Pull request by DBryzz**
[RA-1831](https://issues.openmrs.org/browse/RA-1831): "@Override" should be used on overriding and implementing methods

Added @Override annotation to inherited methods. [Issue description](https://issues.openmrs.org/browse/RA-1831)

Please review and comment where necessary.

### CODE REVIEW

**Review by Wandji69 — APPROVED** [review]
This looks good. Thanks @DBryzz

File: api/src/main/java/org/openmrs/module/coreapps/CoreAppsActivator.java [hunk 1]
```diff
@@ -80,13 +80,15 @@ public static EncounterDiagnosesByObsTagHandler setupEncounterDiagnosesByObsTagH
     /**
 	 * @see ModuleActivator#willRefreshContext()
 	 */
+    @Override
```
Comment (sacull) [inline api/src/main/java/org/openmrs/module/coreapps/CoreAppsActivator.java:83]: At this point, you have 4 spaces instead of one tab.

**Review by sacull — CHANGES_REQUESTED** [review]

File: api/src/main/java/org/openmrs/module/coreapps/CoreAppsActivator.java (hunk 1 above)
Comment (DBryzz) [reply api/src/main/java/org/openmrs/module/coreapps/CoreAppsActivator.java]: updated

**Comment by sacull** [comment]
I only meant line 83 to standardize the writing style in the indicated passage:
image

I am aware that there are tabs in one part of the file and spaces in the other. I am inclined to believe that tabs are accepted in Java as proper, however, correcting all spaces for tabs, in the entire file, is a bit off the ticket.

**Comment by DBryzz** [comment]
Hello @sacull 
I did some changes please check it

**Comment by sacull** [comment]
When you modify the elements mentioned in the ticket, try not to remove and modify elements that are not included in the ticket in the future. Example below:
image

The red color is visible and attracts attention when the ticket says that you will only add something. Even if it is whitespace manipulation.

**Comment by DBryzz** [comment]
@sacull 
The changes I did last time seemed to have affected the entire file whereas what  I wanted was to change only the four spaces found on line 83 to a tab.
I didn't know exactly how many spaces were converted to tabs so I copied the entire content of the file from the master branch and then replaced the content of the file in my branch with what I copied.
That is why it appears the way it is.

If it doesn't meet your expectations please tell me what I can do to fix it.

**Review by sacull — APPROVED** [review]

**Comment by sacull** [comment]
I know that with whitespace IDEs can make surprises. ;)

**Comment by DBryzz** [comment]
Understood.
Thanks for your comments :)

**Pull request merged**